src/main/java/org/opentripplanner/routing/error/RoutingValidationException.java

```diff
@@ -41,4 +43,32 @@ public List<RoutingError> getRoutingErrors() {
   public String getMessage() {
     return routingErrors.stream().map(Objects::toString).collect(Collectors.joining("\n"));
   }
+
+  public boolean isFromLocationNotFound() {
+    return (
+      routingErrors != null &&
```

Comment:
This method is identical to the one below apart from one parameter. Please make it DRY.

Reply:
done